.travis.yml

```diff
@@ -17,6 +17,7 @@ before_install:
   # Override Xenial's default Java version (github.com/travis-ci/travis-ci/issues/10290)
   - export PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
   - export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+  - sudo apt-get install openssl
```

Comment:
Instead of doing this, just add `openssl` to the list that's already present under `addons: apt: packages:`